provisioning/templates/dashboard.html.j2

```diff
@@ -0,0 +1,294 @@
+                <a href="http://{{ servername }}" class="btn btn-success btn-xs" role="button">Open</a>
+                <a href="http://docs.drupalvm.com/en/latest/extras/xhprof/" target="_blank" class="btn btn-info btn-xs" role="button">Documentation</a>
+              </td>
+            </tr>
+          {%- endif -%}
+        {%- endif %}
+      </table>
+    {%- endmacro -%}
+
+    {%- macro sectionDatabaseConnection() -%}
+      <table class="table">
+        <tr>
+          <th>MySQL Hostname</th>
+          <td><code>127.0.0.1</code></td>
+        </tr>
+        <tr>
+          <th>MySQL Username</th>
+          <td><code>{{ mysql_port }}</code></td>
+        </tr>
+        <tr>
+          <th>MySQL Password</th>
+          <td><code>{{ mysql_root_password }}</code></td>
+        </tr>
+        <tr>
+          <th>SSH Hostname</th>
+          <td><code>{{ vagrant_ip }}</code></td>
+        </tr>
+        <tr>
+          <th>SSH Username</th>
+          <td><code>{{ vagrant_user }}</code></td>
+        </tr>
+        <tr>
+          <th>SSH Private Key</th>
+          <td><code>~/.vagrant.d/insecure_private_key</code></td>
+        </tr>
+      </table>
+    {%- endmacro -%}
+
+    {%- macro sectionDatabaseList() -%}
+      <table class="table">
+        <thead>
+          <tr>
+            <th>Database name</th>
+            {% if 'adminer' in installed_extras -%}
+              <th></th>
+            {%- endif %}
+          </tr>
+        </thead>
+        <tbody>
+          {% for database in mysql_databases -%}
+            <tr>
+              <td><code>{{ database.name }}</code></td>
+              {% if 'adminer' in installed_extras -%}
+              <td class="text-right">
+                <a href="http://{{ getServernameFromDocroot(adminer_install_dir) }}/{{ adminer_install_filename }}?username={{ mysql_root_username }}&db={{ database.name }}" class="btn btn-success btn-xs" role="button">Adminer</a>
+              </td>
+              {%- endif %}
+            </tr>
+          {%- endfor %}
+        </tbody>
+      </table>
+    {%- endmacro -%}
+
+    {%- macro sectionDatabaseUserList() -%}
+      <table class="table">
+        <thead>
+          <tr>
+            <th>Username</th>
+            <th>Password</th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for user in mysql_users -%}
+            <tr>
+              <td><code>{{ user.name }}</code></td>
+              <td><code>{{ user.password }}</code></td>
+            </tr>
+          {%- endfor %}
+        </tbody>
+      </table>
+    {%- endmacro -%}
+
+    <div class="container">
+
+      <div class="row">
+        <div class="col-md-12">
+          <section class="jumbotron">
+            <div class="row">
+              <div class="col-md-7">
+                <h1>Welcome!</h1>
+                <p><a href="https://github.com/geerlingguy/drupal-vm">Drupal VM</a> is A VM for local Drupal development, built with Vagrant + Ansible.</p>
```

Comment:
Typo: `A VM`. Appears in https://github.com/geerlingguy/drupal-vm/blob/master/docs/index.md as well.